TAT.homework5_2/IpReader.java

```diff
@@ -0,0 +1,62 @@
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Provides reading IPs from command line or defined file
+ * @author Yury Suponev
+ */
+public class IpReader {

```

Comment:
It's better to split to three classes - base abstract IpReader, specific implementations FileIpReader and CommandLineArgsIpReader